connector/MongoDBAtlasODBC.pq

```diff
@@ -11,8 +10,7 @@ AtlasSQLLink = "https://www.mongodb.com/atlas/sql";
 [DataSource.Kind = "MongoDBAtlasODBC", Publish = "MongoDBAtlasODBC.Publish"]
 shared MongoDBAtlasODBC.Contents = Value.ReplaceType(MongoDBAtlasODBCImpl, MongoDBAtlasODBCType);
 
-[DataSource.Kind = "MongoDBAtlasODBC"]
-shared MongoDBAtlasODBC.Query = Value.ReplaceType(MongoDBAtlasODBCQueryImpl, MongoDBAtlasODBCQueryType);
+Version = "1.0.0";

```

Comment:
I don't see this `Version` being used.

Reply:
removed since we went with a different way to do it